__tests__/app.test.js

```diff
@@ -331,6 +331,51 @@ describe('PATCH /api/articles/:article_id', () => {
 });
 
 
+describe("GET /api/articles/:topic/topics", () => {
+  test("should return status code 200 and an object containing the article of the given id", () => {
+    const topicQuery = "mitch";
+    return request(app)
+      .get(`/api/articles/topics/${topicQuery}`)
```

Comment:
hi Cam, please look up queries.  You can reuse your getarticles endpoint, as queries can attach to the req.query property on the request.  so the endpoint should look like /api/articles?topics=paper